spoolman/api/v1/spool.py

```diff
@@ -159,10 +159,10 @@ async def create(  # noqa: ANN201
     body: SpoolParameters,
 ):
     if body.remaining_weight is not None and body.used_weight is not None:
-        return JSONResponse(
-            status_code=400,
-            content={"message": "Only specify either remaining_weight or used_weight."},
-        )
+       return JSONResponse(
+           status_code=400,
+           content={"message": "Only specify either remaining_weight or used_weight."},
+       )
```

Comment:
Undo this change

Reply:
sorry, i thought i did, apparently i removed the space as well as the commend